controllers/update_status.go

```diff
@@ -299,6 +299,23 @@ func (s UpdateStatus) Reconcile(r *FoundationDBClusterReconciler, context ctx.Co
 		status.FailingPods = nil
 	}
 
+	if len(cluster.Spec.LockOptions.DenyList) > 0 && cluster.ShouldUseLocks() && status.Configured {
+		lockClient, err := r.getLockClient(cluster)
+		if err != nil {
+			return false, err
+		}
+		denyList, err := lockClient.GetDenyList()
+		if err != nil {
+			return false, err
+		}
+		if len(denyList) == 0 {
+			denyList = nil
+		}
+		status.Locks.DenyList = denyList
+	} else {
+		status.Locks.DenyList = nil
```

Comment:
Since we don't set this value anywhere above, shouldn't this be the default value?

Reply:
Also the "new" `equality.Semantic.DeepEqual` method (should) take care of this:

```
An empty slice *is* equal to a nil slice for our purposes; same for maps.
```